ssc/cmod_battery.h

```diff
@@ -211,32 +211,42 @@ struct batt_variables
 	double batt_cycle_cost;
 };
 
+struct batt_time_settings
+{
+    int current_year;
+    int current_hr;
+    int current_step;
+
+};
 
 struct battstor
 {
 	/// Pass in the single-year number of records
-	battstor( compute_module &cm, bool setup_model, size_t nrec, double dt_hr, batt_variables *batt_vars=0);
+	battstor(var_table &vt, bool setup_model, size_t nrec, double dt_hr, const std::shared_ptr<batt_variables> batt_vars_in=0);
+
+    battstor(const battstor& orig);
+
```

Comment:
These are the unresolved externals from the Test project.

Reply:
This should be fixed now.